src/js/components/Button/Button.tsx

```diff
@@ -73,7 +73,7 @@ const Button: React.FC<ButtonAttributes & Props> = ({
       disabled={isDisabled}
     >
       {children}
-      {icon && <Icon name={icon} theme={iconTheme} />}
+      {icon && <Icon name={icon} theme={iconTheme} withHover={!isDisabled} />}
```

Comment:
Just making sure that the icon won't change colour on hover if the button is disabled